plugin/Controller/EstateListBase.php

```diff
@@ -126,6 +127,23 @@ public function getGeoSearchBuilder(): GeoSearchBuilder;
 
 	public function setGeoSearchBuilder(GeoSearchBuilder $pGeoSearchBuilder);
 
+	/**
+	 *
+	 * @return GeoSearchBuilderSimilarEstates
+	 *
+	 */
+
+	public function getGeoSearchBuilderSimilarEstates(): GeoSearchBuilderSimilarEstates;
+
+
+	/**
+	 *
+	 * @param GeoSearchBuilderSimilarEstates $pGeoSearchBuilder
+	 *
+	 */
+
+	public function setGeoSearchBuilderSimilarEstates(GeoSearchBuilderSimilarEstates $pGeoSearchBuilder);
```

Comment:
Why cant `setGeoSearchBuilder` be used?

Reply:
I can't use "setGeoSearchBuilder".Because I don't know where and how it was used. I'm afraid I'll overwrite your old code.
If you still want to use it I will fix it